_licenses/mulanpsl-2.0.txt

```diff
@@ -0,0 +1,226 @@
+---
+title: Mulan Permissive Software License，Version 2
```

Comment:
Regular comma
```suggestion
title: Mulan Permissive Software License, Version 2
```

Reply:
Looks like it is the full width comma in the source, and may be intentional https://en.wikipedia.org/wiki/Comma#Languages_other_than_Western_European

Reply:
The title here needs to match the name in https://spdx.org/licenses/licenses.json -- if that value is incorrect it should be fixed in https://github.com/spdx/license-list-XML/blob/master/src/MulanPSL-2.0.xml